docs/guide/tutorial-7-using-ros.md

```diff
@@ -23,10 +23,10 @@ rosdep update
 For more information or to install it on another platform please read [http://wiki.ros.org/ROS/Installation](http://wiki.ros.org/ROS/Installation).
 Unless you need older version for some other application, you should choose the latest distribution (Melodic Morenia or Kinetic Kame).
 
-> **Note**: If you never used the ROS framework before, it is strongly recommended to follow some tutorials from: [http://wiki.ros.org/ROS/Tutorials](http://wiki.ros.org/ROS/Tutorials).
+If you never used the ROS framework before, it is strongly recommended to follow some tutorials from: [http://wiki.ros.org/ROS/Tutorials](http://wiki.ros.org/ROS/Tutorials).
 These tutorials will also help you set up your ROS environment and initialize your catkin workspace.
 
-### "webots\_ros" Package Installation
+### `webots_ros` Package Installation
```

Comment:
The rendering is not nice:
![Capture du 2019-03-21 09-33-14](https://user-images.githubusercontent.com/2461619/54740694-690e7200-4bbc-11e9-898a-eb8c84882f12.png)